pkg/config/prow-config-documented.yaml

```diff
@@ -1355,6 +1355,15 @@ scheduler:
         # configured to in the first place.
         mappings:
             "": ""
+    rest_by_job:
+        # Cache is the cache configuration for the REST by job scheduling strategy
+        cache:
+            # CacheCleanupInterval is the interval to clean up the cache
+            cache_cleanup_interval: 0s
```

Comment:
Since its already under the `cache` stanza, we can just name it as  `cleanup_interval`